src/components/Resources/index.test.js

```diff
@@ -64,9 +99,18 @@ describe('Resources', () => {
     );
 
     await waitFor(() => screen.getByText('Resources'));
+
     expect(axiosMock.get).toHaveBeenCalledTimes(1);
     expect(axiosMock.get).toHaveBeenCalledWith(url);
     expect(screen.getByText('Resources')).toBeInTheDocument();
     expect(screen.getByText('Here garden school full.')).toBeInTheDocument();
+
+    const searchElement = screen.getByRole('searchbox');
+
+    await userEvent.type(searchElement, 'betty');
+    expect(searchElement).toHaveAttribute('value', 'betty');
+
+    fireEvent.keyDown(searchElement, { key: 'Enter', code: 'Enter' });
+    expect(axiosMock.get).toHaveBeenCalledTimes(1);
```

Comment:
Additionally, we need tests around how we render responses from Axios. That's currently unclear without booting up code itself locally.

Reply:
👀  Can you say more? I was struggling with this too, and thought we should just be mocking the data... the deployed site can talk to an api-staging.codebuddies.org, but have yet to set that up.